packages/block-editor/src/components/iframe/index.js

```diff
@@ -54,6 +54,11 @@ function styleSheetsCompat( doc ) {
 		);
 
 		if ( isMatch && ! doc.getElementById( ownerNode.id ) ) {
+			// eslint-disable-next-line no-console
+			console.error(
+				`Stylesheet ${ ownerNode.id } should be added as an editor style.`,
```

Comment:
Block styles can also end up here as well right? Should we include that in the message? If it's a theme style sheet, make it an editor style, if it's a block style, assign it to the block style property?

Reply:
No, theme styles are handled through the settings, so they're always correctly loaded.

Reply:
Oh, right. For blocks it should be `package.json`. I'll link to the relevant documentation.